spec/JSON-RPC.md

```diff
@@ -1264,6 +1265,34 @@ No parameters
 
 [Back to **List of methods**](#list-of-methods)
 
+## chain_getPendingTransactionsCount
+Returns a count of the transactions that are in the current transaction queue.
+
+### Params
+No parameters
+
+### Returns
+`Count`
```

Comment:
Let's use `U64`

Reply:
How about using `number`?
I think it gives enough precision for the number of transactions.

Reply:
I've made the changes to `number`.